README.md

```diff
@@ -1,10 +1,6 @@
-<h1>
-hvPlot <img src="/doc/_static/logo.png" width="50" height="50">
-</h1>
+# hvPlot <img src="doc/_static/logo.png" style="width:2em;margin-bottom:-15px">
```

Comment:
I think it's an advantage to have absolute links on the README, so that it renders correctly in different contexts, .e.g on PyPi.

```suggestion
# hvPlot <img src="https://github.com/holoviz/hvplot/blob/master/doc/_static/logo.png" style="width:2em;margin-bottom:-15px">
```